File: src/codes/Users_Manual/stm_prop.py
~from pyautd3 import Hz, FociSTM
~import numpy as np
foci = [np.array([0.0, 0.0, 0.0]), np.array([0.0, 0.0, 0.0])]
stm = FociSTM.from_sampling_config(1 * Hz, foci)
f = stm.freq                  # -> 1Hz
p = stm.period                # -> 1s
fs = stm.sampling_config.freq # -> 2Hz

File: src/codes/Users_Manual/stm_prop.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
let stm = FociSTM::from_sampling_config(1 * Hz, [Vector3::zeros(), Vector3::zeros()]);
let f = stm.freq();                    // -> 1Hz
let p = stm.period();                  // -> 1s
let fs = stm.sampling_config().freq(); // -> 2Hz
# Ok(())
# }